.github/workflows/pr-cleanup.yml

```diff
@@ -0,0 +1,51 @@
+name: CI On Pull Request Closure
```

Comment:
This workflow was purposely not added. This would mess up the workflow in rasa-private which always fetches the latest image from aws sdk repo [here](https://github.com/RasaHQ/rasa-private/blob/main/.github/workflows/continous-integration.yml#L253). 
Also I have configured the AWS repo to clean up images every 2 days so there wont be older images left behind in the aws repo.

Reply:
Oh, yes, I forgot about the cleanup rule we have in AWS ECR.